ui/app/styles/clinical/_visit.scss

```diff
@@ -631,6 +631,16 @@ legend.mylegend {
     text-align: center;
 }
 
+.provider-info {
+    table td{
+        vertical-align: top;
+        padding-bottom: 15px;
+        line-height: 1.5;
+        table-layout: fixed;
+        width: 50%;
```

Comment:
Since we are showing label + value, it will be easier to view if we show only 2 items per row on mobile(for screen widths upto 768px) ---

| Label | Value |
|-------|-------|
| Label | Value |

| Label<br>Value | Label<br>Value |
|----------------|----------------|

Reply:
done. pls refer card for screenshots